website/pages/docs/k8s/compatibility.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+page_title: Compatibility Matrix
+sidebar_title: Compatibility Matrix
+description: Compatibility Matrix for Consul Kubernetes and Consul
+---
+
+# Compatibility Matrix for Consul Kubernetes and Consul 
+
+Consul Kubernetes is released through Consul Helm. For every release of Consul Kubernetes, a new version of the Consul Kubernetes 
+Helm chart and Consul Kubernetes binary is released through the HashiCorp Helm repository. The general best practice is to upgrade
+the Helm chart for upgrades of Consul Kubernetes since Consul Kubernetes binaries may have dependencies with certain versions of
+the Helm chart. 
+
+## Supported Versions
+
+| Consul Version      | Compatible Consul Helm Versions  |
+| ------------------- | -------------------------------- |
+| 1.8.x               | v0.22 - v0.24.1                  |
+| 1.7.x               | v0.17 - v0.21                    |
+| 1.6.x               | v0.10 - v0.16.2                  |
```

Comment:
I looked at the underlying values.yaml files for these releases and that this matches exactly.

I think the versions should not be prefixed with `v` since for helm they won't have a v prefix, e.g. `helm install --version 0.22.0`. Also instead of using `0.22` we should use `0.22.0` since `0.22` isn't actually an installable version (and for the others)

Reply:
Resolved, thanks!